lib/ts/base/src/ts/twxplore/lib/base/util/logging/LoggerContext.ts

```diff
@@ -1,5 +1,9 @@
 import {ConsoleLogger} from "./ConsoleLogger";
 import {Logger} from "./Logger";
 import * as React from "react";
+import {Environment} from "../../Environment";
+import {NopLogger} from "./NopLogger";
 
-export const LoggerContext = React.createContext<Logger>(new ConsoleLogger());
+export const LoggerContext = React.createContext<Logger>(
+  Environment.development ? new ConsoleLogger() : new NopLogger()
```

Comment:
You need a similar statement in main.tsx.
Also, presumably you'll want to pass the logger in main.tsx to your store instances so the reducer has a logger.

Reply:
The line above is just the default value if no logger is specified, but you see in main.tsx that it always is, so this line could probably have stayed new ConsoleLogger() (or even new NopLogger).